ios/AppLovinMAX.m

```diff
@@ -513,6 +579,17 @@ - (instancetype)init
     self.sdk.targetingData.interests = interests;
 }
 
+RCT_EXPORT_METHOD(getTargetingDataInterests:(RCTPromiseResolveBlock)resolve :(RCTPromiseRejectBlock)reject)
+{
+    if ( !self.sdk )
+    {
+        resolve(self.targetingInterestsToSet);
```

Comment:
Should this be reject?

Reply:
Before the initialization, we cache the value, so this should be resolve.